src/components/Navbar.jsx

```diff
@@ -0,0 +1,139 @@
+import PropTypes from "prop-types";
+import { useEffect, useRef, useState } from "react";
+import { BsTranslate } from "react-icons/bs";
+import { RxHamburgerMenu } from "react-icons/rx";
+
+import logoSrc from "../assets/logo.svg";
+
+const Navbar = ({ className, sections, lngs }) => {
+  const [isTranslationMenuOpen, setIsTranslationMenuOpen] = useState(false);
+  const [isHamburgerMenuOpen, setIsHamburgerMenuOpen] = useState(false);
+  const translationMenuRef = useRef(null);
+  const hamburgerMenuRef = useRef(null);
+
+  useEffect(() => {
+    const handleClickOutside = (event) => {
+      if (
+        !translationMenuRef.current?.contains(event.target) &&
+        !hamburgerMenuRef.current?.contains(event.target)
+      ) {
+        setIsTranslationMenuOpen(false);
+        setIsHamburgerMenuOpen(false);
+      }
+    };
+
+    document.addEventListener("mousedown", handleClickOutside);
+
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, [translationMenuRef, hamburgerMenuRef]);
+
+  return (
+    <nav className={className}>
+      <div className="container flex justify-between items-center px-6 text-white rounded-full shadow-2xl bg-black/75 font-heading">
+        <div className="my-3 ml-12 sm:ml-16">
+          <img
+            alt="Arise logo"
+            className="absolute -translate-x-16 sm:-translate-x-20 -translate-y-2/5 size-16 sm:size-20"
+            src={logoSrc}
+          />
+          <p className="font-bold text-orange-500 uppercase sm:text-xl">
+            Arise
```

Comment:
missing the word 'Movement' in the title. should be Arise Movement

Reply:
checked with design team to remove "Movement" to give more space to the sections